### PR TITLE
docs: simplify strandedness instructions for bedgraph/wig

### PR DESCRIPTION
**Pull request by ovesh**

### CODE REVIEW

File: genome_kit/genome_track.py [hunk 1]
```diff
@@ -76,6 +76,10 @@ def __init__(self, outfile, etype, strandedness, reference_genome, dim=1, resolu
             * ``"single_stranded"``: both strands share the same data. The data is applied in Interval coordinate (reference strand) order.
             * ``"strand_unaware"``: ignores the Interval strand, data is applied in Interval coordinate (reference strand) order.
             * ``"strand_aware"``: data is applied from 5" end to 3" end (sense strand order).
+
+            Negative strand data from bedgraph or wig files must be provided from a separate file. When calling
+            ``set_data_from_bedgraph()`` or ``set_data_from_wig()``, set ``"strand_unaware"`` (``"single_stranded"`` is
```
Comment (s22chan) [inline genome_kit/genome_track.py:81]: use a py::meth reference

File: genome_kit/genome_track.py [hunk 2]
```diff
@@ -76,6 +76,10 @@ def __init__(self, outfile, etype, strandedness, reference_genome, dim=1, resolu
...
+            equivalent when only providing positive strand data).
```
Comment (s22chan) [inline genome_kit/genome_track.py:82]: seems unnecessarily wordy, is the recommendation using two different enums in two scenarios or to always use strand_unware?

Comment (ovesh) [reply genome_kit/genome_track.py:82]: You're right, I'll remove that last part. The recommendation is to always use strand_unaware.

**Review by gkaur — APPROVED** [review]
Looks great! Thanks @ovesh

**Review by s22chan — APPROVED** [review]

**Pull request merged**